src/main/java/gov/nih/nci/evs/api/service/ElasticQueryServiceImpl.java

```diff
@@ -810,7 +811,7 @@ public List<Concept> getMapsets(IncludeParam ip) throws Exception {
             .withPageable(PageRequest.of(0, 10000))
             .build();
 
-    return getResults(query, Concept.class, ElasticOperationsService.MAPPING_INDEX);
+    return getResults(query, Concept.class, ElasticOperationsService.MAPSET_INDEX);
```

Comment:
Are we replacing the MAPPING_INDEX with the MAPSET_INDEX? Do we not need the mapping index anymore or is the mapset index including the data from the mapping index?

Reply:
Mapping index now contains the mappings from the mapsets separately from the rest of the mappings. Both are still used in different contexts. The replacement here was modifying the names of the indexes such that they represented what they hold (mappings index holds the mappings and mapset index holds the mapsets)